lib/factory-helper/name.rb

```diff
@@ -8,14 +8,35 @@ def name
         parse('name.name')
       end
 
-      def first_name; fetch('name.first_name'); end
-      def last_name;  fetch('name.last_name'); end
-      def prefix;     fetch('name.prefix'); end
-      def suffix;     fetch('name.suffix'); end
+      def first_name
+        fetch('name.first_name')
```

Comment:
Would it make sense to randomly fetch `male_name` or `female_name`, with a fallback to `first_name` for locales that don't define those?